src/main/kotlin/com/ekino/oss/gradle/plugin/quality/QualityPlugin.kt

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright (c) 2019 ekino (https://www.ekino.com/)
+ */
+
+package com.ekino.oss.gradle.plugin.quality
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPlugin
+import org.gradle.api.plugins.quality.Checkstyle
+import org.gradle.api.plugins.quality.CheckstyleExtension
+import org.gradle.api.plugins.quality.CheckstylePlugin
+import org.gradle.api.resources.TextResource
+import org.gradle.api.tasks.SourceSetContainer
+import org.gradle.kotlin.dsl.*
+import org.gradle.testing.jacoco.plugins.JacocoPlugin
+import org.gradle.testing.jacoco.plugins.JacocoPluginExtension
+import org.gradle.testing.jacoco.tasks.JacocoReport
+import org.sonarqube.gradle.SonarQubeExtension
+import org.sonarqube.gradle.SonarQubePlugin
+
+class QualityPlugin: Plugin<Project> {
+
+  override fun apply(project: Project) {
+    with(project) {
+
+      // plugins
+      apply<JavaPlugin>()
+      apply<CheckstylePlugin>()
+      apply<JacocoPlugin>()
+      apply<SonarQubePlugin>()
+
+      val sourceSets = the<SourceSetContainer>()
+
+      // configuration
+      configure(listOf(project)) {
+
+        // Checkstyle configuration
+        configure<CheckstyleExtension> {
+          toolVersion = "8.19"
+        }
+
+        tasks.withType<Checkstyle> {
+          config = getTextResource(project, "checkstyle.xml")
+        }
+
+        // Jacoco configuration
+        configure<JacocoPluginExtension> {
+          toolVersion = "0.8.3"
```

Comment:
Same here, not sure about that.

Reply:
I have tested : default version of Jacoco tool for Gradle v5.6.1 is 0.8.4, which is the latest. I will remove this configure block

Reply:
Done